dev-lang/ocaml/ocaml-4.09.0.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+HOMEPAGE="https://ocaml.org/"
+SRC_URI="https://github.com/ocaml/ocaml/archive/${PV}.tar.gz -> ${P}.tar.gz"
+DESCRIPTION="Programming language supporting functional, imperative & object-oriented styles"
+
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+# X flag required by dev-ml/camlimages and others
```

Comment:
but this is then unclear how you utilize this flag other than adding this into `IUSE`

Reply:
I dont utilize it at all, that's the point.

Reply:
As discussed on IRC, this requires a change to the three broken ebuilds.